src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -284,23 +283,15 @@ def add_objectstore(self, spec):
         with self.ignore_409("CephObjectStore '{0}' already exists".format(spec.name)):
             self.rook_api_post("cephobjectstores/", body=rook_os)
 
-    def rm_service(self, service_type, service_id):
-        assert service_type in ("mds", "rgw", "nfs")
-
-        if service_type == "mds":
-            rooktype = "cephfilesystems"
-        elif service_type == "rgw":
-            rooktype = "cephobjectstores"
-        elif service_type == "nfs":
-            rooktype = "cephnfses"
+    def rm_service(self, rooktype, service_id):
 
         objpath = "{0}/{1}".format(rooktype, service_id)
 
         try:
             self.rook_api_delete(objpath)
         except ApiException as e:
             if e.status == 404:
-                log.info("{0} service '{1}' does not exist".format(service_type, service_id))
+                log.info("{0} service '{1}' does not exist".format(rooktype, service_id))
```

Comment:
In other parts of this PR, `log.func(fmt, *args)` is used.

Reply:
I don't want to make big refactorings in the rook module just now.